controllers/utils/init.go

```diff
@@ -31,6 +32,7 @@ func RegisterSchemes(s *runtime.Scheme) {
 	utilruntime.Must(istiosecurityv1beta1.AddToScheme(s))
 	utilruntime.Must(telemetryv1alpha1.AddToScheme(s))
 	utilruntime.Must(maistrav1.SchemeBuilder.AddToScheme(s))
+	utilruntime.Must(knservingv1.AddToScheme(s))
```

Comment:
are we enforcning knative?

Reply:
Nope. This is, indeed, important for RawDeployment.
I think it is OK that the schema is added, regardless. However, it do is important that the webhook is not configured.

I fixed in my latest commit.